plugins/gatsby-theme-iterative-docs/src/utils/shared/sidebar.js

```diff
@@ -23,12 +23,22 @@
 const { titleCase } = require('title-case')
 const sidebar = require('../../../sidebar')
 
+const uppercaseKeywords = ['dvc', 'cml', 'api', 'ml']
```

Comment:
@jorgeorpinel, these are the keywords that will be uppercased. Does the list look good?

Reply:
let's add MLEM, LDB

Reply:
List is now `['dvc', 'cml', 'api', 'mlem', 'ml', 'ldb']` :)